src/components/cards/SightingsCard.jsx

```diff
@@ -83,9 +83,9 @@ export default function SightingsCard({
     return '';
   };
 
-  const encountersWithLocationData = useMemo(
+  const sightingsWithLocationData = useMemo(
     () => {
-      const encsWithFormattedLocations = encounters.map(
+      const encsWithFormattedLocations = sightings.map(
```

Comment:
while we're in the process of naming variables better, makes sense to do this one, too?